lib/system/seqs_v2.nim

```diff
@@ -41,6 +41,15 @@ proc newSeqPayload(cap, elemSize, elemAlign: int): pointer {.compilerRtl, raises
   else:
     result = nil
 
+proc newSeqPayloadUninitialized(cap, elemSize, elemAlign: int): pointer {.compilerRtl, raises: [].} =
+  # Used in `newSeqOfCap()`.
```

Comment:
`newSeqOfCap` should be initialized.

Reply:
@juancarlospaco as I said in the forum, it shouldn't - only capacity is allocated, the user can't access the elements of the seq unless they're `add`ed

Reply:
What happens if a user uses `setLen`?